packages/skeleton/src/lib/utilities/Modal/Modal.svelte

```diff
@@ -183,7 +183,11 @@
 
 	function onPromptSubmit(event: SvelteEvent<SubmitEvent, HTMLFormElement>): void {
 		event.preventDefault();
-		if ($modalStore[0].response) $modalStore[0].response(promptValue);
+		if ($modalStore[0].response) {
```

Comment:
@Mahmoud-zino add a comment just above this line explaining the reason for this change. Something like:

> Ensures input[number] retains numeric type when up/down arrows are pressed in browser.

Once that's add this is good to merge.